test/app/components/plugin/plugin-action-bar.test.js

```diff
@@ -237,7 +237,50 @@ describe('Plugin action bar', () => {
         'edit-preview',
       ]);
 
-      expectEnvPlugin(['preview', 'live']);
+      expectEnvPlugin(['edit', 'preview', 'live']);
+    });
+
+    it('Editor - should display "last edited" description for Source', async () => {
+      sidekickTest
+        .mockFetchEditorStatusSuccess(HelixMockContentSources.SHAREPOINT, HelixMockContentType.DOC)
+        .mockFetchSidekickConfigSuccess(false, false)
+        .mockEditorAdminEnvironment(EditorMockEnvironments.EDITOR)
+        .createSidekick();
+
+      await sidekickTest.awaitStatusFetched();
+
+      await sidekickTest.awaitEnvSwitcher();
```

Comment:
Could just `awaitEnvSwitcher` if you don't care about the result of `awaitStatusFetched`